snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -682,13 +677,11 @@ static void DistinctEx1()
         static void ElementAtEx1()
         {
             // <Snippet28>
-            string[] names =
-                { "Hartono, Tommy", "Adams, Terry", "Andersen, Henriette Thaulow",
-
-                    "Hedlund, Magnus", "Ito, Shu" };
-            Random random = new Random(DateTime.Now.Millisecond);
+            string[] names = [ "Hartono, Tommy", "Adams, Terry", "Andersen, Henriette Thaulow",
+                    "Hedlund, Magnus", "Ito, Shu" ];
+            Random random = new(DateTime.Now.Millisecond);
 
-            string name = names.ElementAt(random.Next(0, names.Length));
+            string name = names[random.Next(0, names.Length)];

```

Comment:
Here again, I'd revert this change. The *purpose* of the snippet is to demonstrate the `ElementAt` method. Removing it defeats the purpose.